combustion-android-ble/src/main/java/inc/combustion/framework/ble/device/UartBleDevice.kt

```diff
@@ -63,13 +63,16 @@ internal open class UartBleDevice(
                 return waiting.get()
             }
 
-        fun handled(result: Boolean, data: Any?, reqId: UInt? = null) {
+        // Returns true if the message was handled, false otherwise
```

Comment:
Maybe semantics but just want to confirm - what determines handled? if current `requestId` matches `reqId` or if callback also exists?

Reply:
> what determines handled? if current requestId matches reqId or if callback also exists?

I think that is exactly what we want. @angrygorilla I believe right now the code will only return true if the callback exists **and** the request id matches, since the callback might not always be set the function should still retrun true if the request id matches (and is also set).

Reply:
great catch. Ill update it